docs/Guide/utilities/Discord_Utilities/UsefulRegexes.mdx

```diff
@@ -162,4 +162,4 @@ This regex matches any url starting with wss/ws
 
 This regex captures a Discord Webhook ID and token from a webhook url
 
-`Raw`: `/(?<url>^https:\/\/(?:(?:canary|ptb).)?discordapp.com\/api\/webhooks\/(?<id>\d+)\/(?<token>[\w-]+)\/?$)/`
+`Raw`: `/(?<url>^https:\/\/(?:(?:canary|ptb).)?discord.com\/api\/webhooks\/(?<id>\d+)\/(?<token>[\w-]+)\/?$)/`
```

Comment:
```suggestion
`Raw`: `/(?<url>^https:\/\/(?:(?:canary|ptb).)?discord(?:app)?.com\/api(?:\/v\d+)?\/webhooks\/(?<id>\d+)\/(?<token>[\w-]+)\/?$)/`
```

The raw has to represent what is defined at https://github.com/sapphiredev/utilities/blob/main/packages/discord-utilities/src/lib/regexes.ts#L145